code/game/jobs/job/upp.dm

```diff
@@ -0,0 +1,324 @@
+/datum/job/upp
+	special_role = "UPP"
+	comm_title = "UPP"
+	faction = "Union of Progressive People"
+	supervisors = "the team leader"
+	idtype = /obj/item/card/id
+	skills_type = /datum/skills/pfc/crafty
+	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
+
+	generate_entry_conditions(mob/living/carbon/human/H)
```

Comment:
Here as well:
`/datum/job/upp/generate_entry_conditions(mob/living/carbon/human/H)`
Then fix indentation. The whole file is full of this kind of instances.